.eslintrc

```diff
@@ -7,6 +7,7 @@
   ],
   "env": {
     "browser": true,
+    "es6": true,
```

Comment:
what is this needed? I'm checking `terra-core` and noticing they don't set this? cc: @bjankord